readme.md

```diff
@@ -9,7 +9,44 @@ Brushing teeth too soon after eating or drinking can soften the tooth enamel, es
 
 Breathing deeply in moments of stress, or anytime during the day, brings many benefits such as better circulation, decreased anxiety and reduced 
 blood pressure1.
+#random
+here are some random health facts:
+
+Drinking at least five glasses of water a day can reduce your chances of suffering from a heart attack by 40%1.
+
+Repeatedly using a plastic water bottle can release chemicals into your water1.
+
+Brushing teeth too soon after eating or drinking can soften the tooth enamel, especially if you’ve just been eating or drinking acidic foods1.
+
+Breathing deeply in moments of stress, or anytime during the day, brings many benefits such as better circulation, decreased anxiety and reduced 
+blood pressure1.
+
+<---jnshdf--->

```

Comment:
The partition "<---jnshdf--->" could be replaced with a more meaningful title or a markdown horizontal line for better readability. [medium]